### ISSUE TITLE
Buffer graphics conversion.

### ISSUE THREAD

**Issue by FluBBaOfWard** [FluBBaOfWard/Sphinx#1]
Buffer graphics conversion to get rid of glitches.

**Comment by LegoWasTaken** [comment]
This is an awesome concept. Are any games playable on it yet? Would be a pretty nice way to play the landscape WS games

**Comment by FluBBaOfWard** [comment]
Yes it plays a lot of games, though you'd want to use a Nintendo DSi for the speed and memory.
https://github.com/FluBBaOfWard/NitroSwan
There is a thread over on GBATEMP https://gbatemp.net/threads/nitroswan-a-wonderswan-emulator-for-nds.604718/ if you want to discuss further.
